my/lessons/basics/composition.md

```diff
@@ -6,7 +6,11 @@ order: 8
 lang: my
 ---
 
+<<<<<<< HEAD
```

Comment:
@alkaha there appear to be a bunch of merge conflicts in this file, can you look at resolving them?